.github/workflows/builds.yml

```diff
@@ -22,6 +22,9 @@ jobs:
       uses: cake-build/cake-action@v1.4.0
       with:
         target: Test
+
+    - name: EditorConfig-Action
+      uses: zbeekman/EditorConfig-Action@v1.1.1
```

Comment:
and add variable to forced off to avoid unintentionally enabling when upgrading to new version.

Reply:
could you elaborate?